resources/views/admin/carpool_drivers/create.blade.php

```diff
@@ -0,0 +1,61 @@
+<x-admin-app-layout>
+    <x-slot name="title">
+        Add Carpool Driver
+    </x-slot>
+
+    <x-status-message />
+
+    <!-- Back Button -->
+    <div class="px-4 py-2 mt-6 rounded max-w-max bg-fuchsia-600 hover:bg-fuchsia-700">
+        <a href="{{ route('admin.carpool_drivers.index') }}" class="flex items-center justify-center text-white">
+            <i class="mr-2 fas fa-arrow-left"></i>
+            Back to Carpool Drivers List
+        </a>
+    </div>
+
+    <div class="container grid w-3/5 px-6 mx-auto">
+        <div class="items-center p-4 my-6">
+            <form method="post" action="{{ route('admin.carpool_drivers.store') }}">
```

Comment:
We need to add a dropdown field with users that do not have roles, so that we can assign them the carpool driver role. Carpool drivers are part of the system's users.